package.json

```diff
@@ -67,15 +68,20 @@
   },
   "dependencies": {
     "@types/react": "^16.0.10",
+    "apollo-client-preset": "^1.0.5",
     "axios": "^0.16.2",
     "cli-table": "^0.3.1",
     "colors": "^1.1.2",
     "dateformat": "^3.0.2",
     "ejs": "^2.5.7",
+    "emotion": "^8.0.12",
```

Comment:
Why?

Reply:
Needed for the loading icon I'm using on the `SearchResults`.

Path: `.../core/icons/PropagateLoader.js`

Reply:
Ugh. No way to use other icon?

Reply:
Why not rewrite code in JSS?